testdata/go/v3/memcached-operator/config/manager/manager.yaml

```diff
@@ -45,12 +47,14 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
+        # TODO(user): Configure the resources accordingly based on the project requirements.
+        # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
           limits:
-            cpu: 200m
-            memory: 100Mi
+            cpu: 500m
+            memory: 128Mi
           requests:
-            cpu: 100m
-            memory: 20Mi
+            cpu: 10m
+            memory: 64Mi
```

Comment:
Same question from earlier, where did you get the values that you picked?

Reply:
From the k8s doc example, see the link in the TODO and we are only reduced the requested by default and increased the limits based on the @fgiloux analyses made. 

It is not required to request too much by default. If we do so, this resource is allocated for the operators and we began to consume too much by default. 

The values are defined in KB.